### PR TITLE
Update migration notes heading

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

**Comment by raboof** [comment]
> the new default is the default now, no? :)

yes - the old default behavior is different from the new default behavior, so the default behavior changed?

**Comment by wilhelmy** [comment]
I think I mostly find the parentheses confusing. Either something changes the default behaviour, or the behaviour. I think my ability to articulate has just disappeared due to quarantine. Please disregard.

**Comment by raboof** [comment]
Haha sorry - I'm open to suggestions on a rephrase ;)

File: migration.md [hunk 1]
```diff
@@ -5,9 +5,11 @@ title: Migration
 
 # Migration from (Not)Ion3
 
-The next major release of Notion will be Notion 4. This release will include a
-number of changes that are either not backwards compatible, or change the
-(default) behavior.
+Notion 4 includes a number of changes that are either not backwards compatible,
+or change the (default) behavior.
```
Comment (wilhelmy) [inline migration.md:9]:
```suggestion
or change the default behavior.
```

**Comment by wilhelmy** [comment]
While we're editing the migration notes, didn't you change the `based_on = "*"` back to what it was so that paragraph is obsolete now?

**Comment by raboof** [comment]
> didn't you change the `based_on = "*"` back to what it was so that paragraph is obsolete now?

Yes, I think that's no longer accurate as of #197 - added #282

**Pull request merged**